styles.js

```diff
@@ -1,14 +0,0 @@
-import { createGlobalStyle } from "styled-components";
-
-export default createGlobalStyle`
-  *,
-  *::before,
-  *::after {
-    box-sizing: border-box;
-  }
-
-  body {
-    margin: 0;
-    font-family: system-ui;
-  }
-`;
```

Comment:
why did you remove this file? this should be the GlobalStyles and they are really useful. Please add them to your project.